docs/pages/custom-builds/schema.mdx

```diff
@@ -934,38 +936,23 @@ build:
   href="https://github.com/expo/eas-build/blob/main/packages/build-tools/src/steps/functions/installNodeModules.ts"
 />
 
-#### `eas/get_credentials_for_build_triggered_by_github_integration`
-
-Resolves the credentials for a build triggered by our [GitHub integration](/build/building-from-github/). Call this function if you are using the GitHub integration and your custom build needs to use credentials.
+#### `eas/resolve_build_config`
 
-This is a no-op if the build was not triggered by the GitHub integration. This function is automatically executed by [`eas/build`](#easbuild) function group.
+Resolves and prints the build configuration. If the build has been triggered by the GitHub integration, it will update the current `job` and `metadata` context values. It should be called after installing the dependencies because the config may be influenced by config plugins.
 
-```yaml example.yml
-build:
-  name: Example Android config for a build triggered by GitHub integration
-  steps:
-    - eas/checkout
-    - eas/use_npm_token
-    - eas/install_node_modules
-    # @info #
-    - eas/get_credentials_for_build_triggered_by_github_integration
-    # @end #
-    - eas/configure_eas_update:
-        inputs:
-          throw_if_not_configured: false
-    - eas/inject_android_credentials
-    - eas/configure_android_version
-    - eas/run_gradle
-    - eas/find_and_upload_build_artifacts
-```
```

Comment:
Do we want to give users any usage examples?

Reply:
I hope they won't ever use it manually, only by copying an existing `eas-custom-builds-example` YAML and seeing it there.